packages/orbit-components/src/Icon/Icon.stories.tsx

```diff
@@ -1,129 +1,98 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import IconList from "./IconList";
 import * as Icons from "../icons";
 import { ICON_SIZES, ICON_COLORS } from "./consts";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 
-export default {
-  title: "Icon",
-};
+import OrbitIcon from ".";
 
-export const Default = ({ size, color, source, dataTest, ariaLabel, ariaHidden }) => {
-  const Icon = Icons[source];
-  return (
-    <Icon
-      size={size}
-      color={color}
-      dataTest={dataTest}
-      ariaLabel={ariaLabel}
-      ariaHidden={ariaHidden}
-    />
-  );
-};
+const getIcon = source => Icons[source];
+
+type IconPropsAndCustomArgs = React.ComponentProps<typeof OrbitIcon> & { source: string };
+
+const meta: Meta<IconPropsAndCustomArgs> = {
+  title: "Icon",
+  component: OrbitIcon,
 
-Default.story = {
-  name: "Default",
   parameters: {
     info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-Default.args = {
-  size: ICON_SIZES.MEDIUM,
-  color: ICON_COLORS.PRIMARY,
-  source: "Airplane",
-  dataTest: "test",
-  ariaLabel: "label",
-  ariaHidden: true,
-};
+  args: {
+    size: ICON_SIZES.MEDIUM,
+    color: ICON_COLORS.PRIMARY,
+    source: "Airplane",
+    ariaLabel: "label",
+    ariaHidden: true,
+    customColor: "#ABCDEF",
+  },
 
-Default.argTypes = {
-  size: {
-    options: Object.values(ICON_SIZES),
-    control: {
-      type: "select",
+  argTypes: {
+    size: {
+      options: Object.values(ICON_SIZES),
+      control: {
+        type: "select",
+      },
     },
-  },
-  color: {
-    options: Object.values(ICON_COLORS),
-    control: {
-      type: "select",
+    color: {
+      options: Object.values(ICON_COLORS),
+      control: {
+        type: "select",
+      },
     },
-  },
-  source: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+    source: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const CustomColor = ({ size, customColor, source }) => {
-  const Icon = Icons[source];
+export default meta;
+type Story = StoryObj<IconPropsAndCustomArgs>;
 
-  return <Icon size={size} customColor={customColor} />;
-};
-
-CustomColor.story = {
-  name: "Custom Color",
-  parameters: {
-    info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
+export const Default: Story = {
```

Comment:
I would maybe not set `customColor` in the Default story as there is separate story CustomColor so they are the same now. Here I would set one of the `ICON_COLORS`, WDYT?

Reply:
I am considering disabling `customColor` from `Default` story. When having this prop excluded we can set and see the colors from `color` prop. There is also story `Custom Color` that allows us to set custom hex color.

WDYT? @domihustinova

Reply:
Agree 👍